src/sentry/static/sentry/app/routes.jsx

```diff
@@ -474,7 +474,8 @@ function routes() {
       <Route path="settings/" component={errorHandler(OrganizationGeneralSettingsView)} />
 
       <Route name="Teams" path="teams/">
-        <IndexRedirect to="your-teams" />
+        <IndexRoute name="Teams" component={errorHandler(OrganizationTeams)} />
```

Comment:
@billyvg i'm not sure if this was what you meant or not, so lmk if there's another way i should handle this

Reply:
If it works, then yes that's what I meant :P

I think you should remove `name` prop here to remove it from breadcrumbs